<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-00TCO18I" failures="1" tests="13" name="com.amazon.qa.tests.SignInPageTest" time="145.514" errors="0" timestamp="2025-07-05T17:06:28 IST" skipped="2">
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="7.359"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="18.880"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="6.749"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials2" time="7.445"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials2" time="6.772"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="22.795">
    <skipped/>
  </testcase> <!-- SignInwith_InValidCredentials -->
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials2" time="6.858"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials2" time="6.747"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials2" time="6.776"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="6.947"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_InValidCredentials" time="20.187"/>
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_ValidCredentials" time="8.037">
    <failure message="Failed purposly" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed purposly
at org.testng.Assert.fail(Assert.java:111)
at com.amazon.qa.tests.SignInPageTest.SignInwith_ValidCredentials(SignInPageTest.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
    </failure>
  </testcase> <!-- SignInwith_ValidCredentials -->
  <testcase classname="com.amazon.qa.tests.SignInPageTest" name="SignInwith_ValidCredentials" time="19.962">
    <skipped/>
  </testcase> <!-- SignInwith_ValidCredentials -->
</testsuite> <!-- com.amazon.qa.tests.SignInPageTest -->
